### PR TITLE
Portal/update readme v2

### PR DESCRIPTION
**Pull request by maurineimirandazup**
# Pull Request

## What kind of change does this PR make

- [ ] Major
- [x] Minor
- [ ] Patch

### CODE REVIEW

File: github/add/collaborator/README.md [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
 
-![Example](https://github.com/ZupIT/ritchie-formulas/raw/master/github/add/collaborator/src/docs/github.gif)
-
-## Description
+## Requirements
 
-This formula allows adding a new collaborator by typing only two parameters.
-(collaborator username and repository name)
+- Git
+- Set Github Credentials
```
Comment (fabianofernandeszup) [inline github/add/collaborator/README.md:15]: You can remove this requirement **- Set Github Credentials**

File: github/create/repo/README.md [hunk 2]
```diff
@@ -8,18 +10,10 @@ rit github create repo
...
-
-```bash
-rit set credential
-```
-
-## How to generate personal access token
+- Git
 
-To generate a personal access token click [here](https://github.com/settings/tokens)
+- Set Github Credentials
```
Comment (fabianofernandeszup) [inline github/create/repo/README.md:15]: You can remove this requirement **- Set Github Credentials**

File: github/delete/repo/README.md [hunk 3]
```diff
@@ -1,48 +1,26 @@
...
-
-This Github delete command allows the user to delete a Github PUBLIC or PRIVATE repository on the user domain.
-
-The user has to inform only one input:
-
-- the repository name on Github
-
-Ex: The NAME would be "ritchie-formulas" for URL https://github.com/ZupIT/ritchie-formulas
+- Git
+- Set GITHUB credentials ($ rit set credentials) with USERNAME, TOKEN
```
Comment (fabianofernandeszup) [inline github/delete/repo/README.md:23]: You can remove this requirement **- Set Github Credentials**

File: github/merge/branch/README.md [hunk 4]
```diff
@@ -31,3 +17,19 @@ If push is true
...
+## Command
+
+```bash
+rit github merge branch
+```
+
+## Requirements
+
+- Git
+- Set Git Credentials
```
Comment (fabianofernandeszup) [inline github/merge/branch/README.md:30]: You can remove this requirement **- Set Github Credentials**

File: github/publish/repo/README.md [hunk 5]
```diff
@@ -49,6 +16,17 @@ The user has to inform 4 different kinds of inputs:
...
+## Command
+
+```bash
+rit github publish repo
+```
+
+## Requirements
+
+- Git
+- Set Github Credentials
```
Comment (fabianofernandeszup) [inline github/publish/repo/README.md:28]: You can remove this requirement **- Set Github Credentials**

File: github/search/handbook/README.md [hunk 6]
```diff
@@ -22,18 +24,11 @@ echo '{"provider":"Github", "name":"demo", "version":"2.0.0", "url":"https://git
...
-
-```bash
-rit github search handbook
-```
-
-## description
-
-This formula search a handbook on a github repository
+- [Golang installed](https://golang.org/doc/install)
+- [Ritchie installed](https://docs.ritchiecli.io/v/v2.0-pt/getting-started/installation)
```
Comment (fabianofernandeszup) [inline github/search/handbook/README.md:30]: You can remove this requirement **- [Ritchie installed]**

File: kubernetes/show/health-status/README.md [hunk 7]
```diff
@@ -46,6 +12,18 @@ The user has to inform 2 different kinds of inputs:
...
 
+## Command
+
+```bash
+rit kubernetes show health-status
+```
+
+## Requirements
+
+- Set kubeconfig credentials (`rit set credentials`)
```
Comment (fabianofernandeszup) [inline kubernetes/show/health-status/README.md:23]: You can remove this requirement **-  Set kubeconfig credentials (`rit set credentials`)**

File: kubernetes/show/logs/README.md [hunk 8]
```diff
@@ -1,44 +1,23 @@
...
-```
-
-## Description
-
-This _show logs_ command will continuously show the logs of a running container in the provided cluster. The user has to inform 2 different kinds of inputs:
+## Requirements
 
-- the namespace that the pod is running
-- the name of the pod (or at least a part of it)
+- kubeconfig credentials
```
Comment (fabianofernandeszup) [inline kubernetes/show/logs/README.md:19]: You can remove this requirement **-   kubeconfig credentials**

File: github/add/collaborator/README.md (hunk 1 above)
Comment (maurineimirandazup) [reply github/add/collaborator/README.md]: Done.

File: github/create/repo/README.md (hunk 2 above)
Comment (maurineimirandazup) [reply github/create/repo/README.md]: Done.

File: github/delete/repo/README.md (hunk 3 above)
Comment (maurineimirandazup) [reply github/delete/repo/README.md]: Done.

File: github/publish/repo/README.md (hunk 5 above)
Comment (maurineimirandazup) [reply github/publish/repo/README.md]: Done.

File: github/search/handbook/README.md (hunk 6 above)
Comment (maurineimirandazup) [reply github/search/handbook/README.md]: Done.

File: kubernetes/show/health-status/README.md (hunk 7 above)
Comment (maurineimirandazup) [reply kubernetes/show/health-status/README.md]: Done.

File: kubernetes/show/logs/README.md (hunk 8 above)
Comment (maurineimirandazup) [reply kubernetes/show/logs/README.md]: Done.

**Review by fabianofernandeszup — APPROVED** [review]

**Pull request merged**